lstchain/clean/data_clean.py

```diff
@@ -0,0 +1,40 @@
+import os
+import json
+import numpy as np
+
+__all__ = [
+        'mc_filter',
+        'data_filter'
+]
+
+file = os.path.join(os.path.dirname(__file__),"../data/data_selection_cuts.json")
+
+with open(file) as json_file:
+    data_cut = json.load(json_file)
+
+### TODO: We can generalize for the user to choose which selection cuts to make
+def mc_filter(data):
```

Comment:
These filtering functions are the same? And they have to be the same for data and mc.

Reply:
Yes, they will be. Just that the theta² cut is not applied on real data now. It will be used in gammapy analysis, when selecting the dataset's on_region. Also, I am going to remove this clean submodule and just use filter_events and separate fixed cuts of g/h score and theta cuts, directly in the main script.